### PR TITLE
docs: Minor changes following audit

### PR DESCRIPTION
**Pull request by RichardLitt**
Part of #127

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-  "description": "a GitHub App that closes abandoned issues after a period of inactivity",
+  "description": "A GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.",
   "author": "Brandon Keepers",
   "license": "ISC",
+  "homepage": "https://probot.github.io/apps/stale/",
+  "bugs": "https://github.com/probot/stale/issues",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-plugin",
```
Comment (hiimbex) [inline package.json]: Again, we deprecated the probot-plugin terminology and usage, so I'd prefer not to include it here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+## Just starting out? Looking for how to help?
+
+Use [this search][good-first-issue-search] to find Probot apps that have issues marked with the `good-first-issue` label.
+
+## Bots
+
+We have one bot enabled on this repo:
```
Comment (hiimbex) [inline CONTRIBUTING.md]: nit: bot terminology

File: package.json (hunk 1 above)
Comment (RichardLitt) [reply package.json]: Cool. Again; I would remove it from the GitHub keywords, then. :)

**Comment by hiimbex** [comment]
> Cool. Again; I would remove it from the GitHub keywords, then. :) 

Can you actually delete a GitHub keyword? I think it's technically a topic: https://github.com/topics/probot-plugin So I think the best we can do is remove it from all our projects, which I'll go do now!

Just waiting the +1 from @bkeepers on this PR

**Comment by RichardLitt** [comment]
> Can you actually delete a GitHub keyword? I think it's technically a topic: github.com/topics/probot-plugin So I think the best we can do is remove it from all our projects, which I'll go do now!

That works. :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Use [this search][good-first-issue-search] to find Probot apps that have issues marked with the `good-first-issue` label.
+
+## Apps
+
+We have one app enabled on this repo:
+
+- [Stale](https://probot.github.io/apps/stale/): We use Stale (the app created in this repo) to ensure that conversations here remain relevant. This is for us, the maintainers, so that we don't feel like we've got hundreds of issues to deal with; if you still have an issue, please let us know! We don't want to close issues that are painful for you. Stale just helps us have a bit more breathing space by making sure issues don't pile up forever.
+
+If you're concerned about our apps or feel that they are insensitive in some way, please let us know.
```
Comment (bkeepers) [inline CONTRIBUTING.md]: This section is likely to be the same on 99% of projects in this org, so I'd be up for centralizing it in a future PR.

Comment (RichardLitt) [reply CONTRIBUTING.md]: 👍 Works for me.

**Pull request merged**